scripts/simple_benchmark.py

```
@@ -0,0 +1,95 @@
+    c.execute(
+        "CREATE TABLE appliance (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT NOT NULL UNIQUE)"
+    )
+    c.execute(
+        "CREATE TABLE appliance_availability (id INTEGER PRIMARY KEY AUTOINCREMENT, appliance_id INTEGER NOT NULL, start_time DATETIME NOT NULL, end_time DATETIME NOT NULL)"
+    )
+
+    # Insert crew
+    roles = ["WC", "CM", "CC", "FFC", "FFD", "FFT"]
+    skills_list = ["BA", "LGV", "TTR", "IC", "ERD"]
+
+    now = datetime.now()
```

Ruff (0.15.5)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)